src/server.ts

```diff
@@ -15,15 +15,27 @@ const metricsMiddleware = promBundle({
   includeMethod: true,
   includeUp: true,
 });
-import job from "./utils/dbCronJob"
+import job from "./utils/dbCronJob";
 import rt from "file-stream-rotator";
 
+// log formatting
+morgan.token(
+  "custom",
+  // @ts-ignore
```

Comment:
this is because morgan formatting token is spreading response obj into string for file-stream-rotator, should be fine I guess but let me know if you have any idea to deal with it 😥

Reply:
Maybe you have to install some types? Did you check?

Reply:
Yeah I'd checked it, there isn't for this lib any unfortunately.. I guess something can be done with a interface in `src/types/declaration.d.ts` but can't figure what/how to touch exactly, that's the problem 😁TS is hard!!

good thing is, i'm quite sure destination of this obj is 99.999% string that will only stay inside *.log

bad thing is.. it looks so ugly😂

Reply:
Dont worry about it doesnt matter for now